docs/developer_guide/docker_images.rst

```diff
@@ -0,0 +1,94 @@
+Manually Build Docker Images
+----------------------------
+
+.. note: It is highly recommended to build the docker image on the same basic system or architecture type that you intend to run it on, *i.e.*, AWS Linux AMI 64-bit (x86), as it may experience difficulties running on other radically different systems (like an M1 Mac).
+
+.. note: The NeuroConv docker container comes prepackaged with all required installations, *i.e.*, equivalent to `pip install neuroconv[full]`. As such it is fairly heavy, so be sure that whatever system (or specifically CI environment) you build with has sufficient disk space.
+
+
+Latest Release
+~~~~~~~~~~~~~~
+
+.. code:
+
+    docker build -f neuroconv_latest_release_dockerfile -t neuroconv_latest_release .
+
+
+Dev Branch
+~~~~~~~~~~
+
+Checkout to a specific branch on a local clone, then
+
+.. code:
+
+    docker build -f neuroconv_developer_build_dockerfile -t neuroconv_dev .
+
+
+
+Publish Container to GitHub
+---------------------------
+
+The `LABEL` is the important item here; it determines the host repository on the GitHub Container Registry (GHCR). In each docker file we wish to publish on the GHCR, we will add this label right after the `FROM` clause
+
+.. code:
+
+    FROM PARENT_IMAGE:TAG
+    LABEL org.opencontainers.image.source=https://github.com/catalystneuro/neuroconv
+
+After building the image itself, we can publish the container with
+
+.. code:
+
+    docker tag IMAGE_NAME ghcr.io/catalystneuro/IMAGE_NAME:TAG
+    export CR_PAT="<YOUR GITHUB SECRET TOKEN>"
+    echo $CR_PAT | docker login ghcr.io -u <YOUR GITHUB USERNAME> --password-stdin
+    docker push ghcr.io/catalystneuro/IMAGE_NAME:TAG
+
+.. note: Though it may appear confusing, the use of the `IMAGE_NAME` in these steps determines only the _name_ of the package as available from the 'packages' screen of the host repository; the `LABEL` itself ensured the upload and linkage to the NeuroConv GHCR.
+
+
+
+Running Docker Container on local YAML (Linux)
+----------------------------------------------
+
+You can either perform a manual build locally following the instructions above, or pull the container from the GitHub Container Registry (GHCR) with
+
+.. code:
+
+    docker pull ghcr.io/catalystneuro/neuroconv:latest
+
+and can then run the entrypoint (equivalent to the usual CLI usage) on a YAML specification file (named `your_specification_file.yml`) with
+
+.. code:
+
+    docker run -it --volume /your/local/drive/:/desired/alias/of/drive/ ghcr.io/catalystneuro/neuroconv:latest neuroconv /desired/alias/of/drive/your_specification_file.yml
```

Comment:
Thanks @CodyCBakerPhD, and sorry this has taken be so long to review. This looks great, but as you note in the comment, I think this needs testing. Could you at least create or point to a YAML file that can be used as input for this docker run command?